panels/user-accounts/cc-app-permissions.ui

```diff
@@ -168,7 +168,7 @@
 
         <child>
           <object class="GtkLabel" id="user_installation_label">
-            <property name="visible">True</property>
+            <property name="visible" bind-source="allow_user_installation_switch" bind-property="visible" bind-flags="bidirectional" />
```

Comment:
`bind-flags="default|sync-create"` is a better fit here

Reply:
Oh Ok. I'll read up on those.